config/templates/csv-template.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
This can be a rename of config/templates/must-gather-operator-csv-template.yaml (we can delete the latter in a fup)